vue-app/src/components/list-header.vue

```diff
@@ -32,6 +32,7 @@ export default {
       <h2 class="title">{{ title }}</h2>
     </router-link>
     <button
+      type="button"
```

Comment:
added due to lint rule on accessibility, i.e must have a type, keep or remove? @johnpapa